modules/oxide/src/less/theme/components/editor/editor.less

```diff
@@ -46,6 +47,7 @@
 
 .tox.tox-tinymce--toolbar-sticky-on {
   .tox-editor-header {
+    background-color: @editor-header-sticky-background-color;
```

Comment:
This isn't going to work, as inline mode will always enable sticky headers. So the inline background colour will be used while the toolbar is in view, but then when you scroll and this docks the colour will change to the sticky background colour.

I think we'll need the one variable shared across the two `tox-editor-header` background colour.

Note: I tested this locally by making the inline colour red and then the sticky color green. It swaps from red to green when the toolbar docks to the top of the window.

Reply:
I guess another option would be to move the `.tox.tox-tinymce-inline .tox-editor-header` style from `global.less` to below this declaration so the inline rules always have a higher specificity.

Reply:
I made an adjustment to the logic. Now the inline and sticky toolbars are automatically set if the regular toolbar is transparent. I've also kept the variables separate as I see it would be useful to be able to set them individually as a designer.

Reply:
I think it just complicates things personally, but I don't have enough of an opinion to argue, so marking as resolved.